app/models/course/assessment/answer/scribing.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+class Course::Assessment::Answer::Scribing < ActiveRecord::Base
+  acts_as :answer, class_name: Course::Assessment::Answer.name
+
+  def to_partial_path
+    'course/assessment/answer/scribing/scribing'
+  end
+
+  MAX_ATTEMPTING_TIMES = 1000
```

Comment:
Actually we don't need max attempting times for scribing, can remove this whole portion. 

It is used more for programming questions to get students to attempt on their local machine instead of trying running every time they change the code.